### ISSUE TITLE
Missing libraries have their last letter cut out in the error message

### ISSUE THREAD

**Issue by fvictorio** [nomiclabs/hardhat#1401]
Given a contract like this:

```
import "this-library-does-not-exist";
```

The error message says:

```
Error HH411: The library this-library-does-not-exis, imported from...
```

**Comment by Bidon15** [comment]
The [-1 result from indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf) is causing this cutting, when we execute the slicing.

Possible fix: https://github.com/nomiclabs/hardhat/pull/1405

It is a pleasure to read and have some hands-on experience on what is done under the hood of HH 🧐

**Comment by fvictorio** [comment]
Released in `hardhat@2.3.0`. Thanks @Bidon15!
